docs/plans/2026-02-07-fix-absolute-performance-optimization-plan.md

```
@@ -0,0 +1,81 @@
+---
+---
+title: fix: Absolute performance optimization (startup + schedule)
+type: fix
+date: 2026-02-07
+---
+# Fix: Absolute performance optimization (startup + schedule)
```

markdownlint-cli2 (0.20.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)